cmd/syncthing/gui.go

```diff
@@ -270,10 +269,12 @@ func (s *apiService) Serve() {
 	postRestMux.HandleFunc("/rest/system/restart", s.postSystemRestart)        // -
 	postRestMux.HandleFunc("/rest/system/shutdown", s.postSystemShutdown)      // -
 	postRestMux.HandleFunc("/rest/system/upgrade", s.postSystemUpgrade)        // -
-	postRestMux.HandleFunc("/rest/system/pause", s.postSystemPause)            // device
-	postRestMux.HandleFunc("/rest/system/resume", s.postSystemResume)          // device
 	postRestMux.HandleFunc("/rest/system/debug", s.postSystemDebug)            // [enable] [disable]
 
+	// Deprecated
+	postRestMux.HandleFunc("/rest/system/pause", s.deprecatedMakePauseHandler(true))   // device
```

Comment:
Well because they are values that live in the config (oppose to being some runtime values), to me what we have now makes perfect sense.
Also, I am perfectly fine to have one and one way only to set these, oppose to two, one via config (as that's where they need to live sadly, in order to persist), and one via pause unpause calls.